src/main/scripts/org/reaktivity/specification/nukleus/oauth/streams/proxy/request.with.too.many.scopes.with.signed.jwt.rs256.forwarded/connect/client.rpt

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property oauthToken ${oauth:jwt("ES256")
+                           .claim("iss", "test issuer")
+                           .claim("scope", "scope1")
+                           .claim("scope", "scope2")
+                           .sign()}
+
+connect await ROUTED_PROXY
+        "nukleus://streams/target#0"
+        option nukleus:authorization 0x0001_000000000000L
```

Comment:
Ensure `authorization` is enforcing expected bits after changing scenario to that described above, i.e. `0x0001_000000000005L`.